Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -287,10 +295,25 @@ class CredentialsManagerSpec: QuickSpec {
                 let credentials = Credentials(accessToken: AccessToken, tokenType: TokenType, idToken: ExpiredToken, refreshToken: nil, expiresIn: Date(timeIntervalSinceNow: ExpiresIn))
                 expect(credentialsManager.hasExpired(credentials)).to(beTrue())
             }
-            
+
         }
-        
-        describe("validity with id token") {
+
+        describe("scope") {
+
+            it("should return true when the scope has changed") {
+                let credentials = Credentials(scope: "openid email")
```

Comment:
since tests are also documentation, I'd swap these scope values so we don't show that a token could have its scope increased.

Reply:
Sorry, I don't follow, can you please expand?

Reply:
swap the scope values
```java
let credentials = Credentials(scope: "openid email profile")
expect(credentialsManager.hasScopeChanged(credentials, than: "openid email")).to(beTrue())
```

Reply:
I believe here Lucho is saying, you can only down scope e.g reduce scopes you can't increase them. So,  let's not show up scoping behaviour in tests. Unless there it's a test to show that it doesn't work.

Reply:
Thanks, that makes sense. Changed in https://github.com/auth0/Auth0.swift/pull/399/commits/04ffa990458113d1f2963b18bb82d2fc82d062c2.